filament/backend/src/vulkan/VulkanCommands.cpp

```diff
@@ -33,453 +33,504 @@ using namespace utils;
 
 namespace filament::backend {
 
+    namespace {
```

Comment:
The spacing is different for this file.  

the base classes and namespace should have no spacing in front.